docs/gpio.md

```diff
@@ -15,65 +15,79 @@ There are 4 modes:
 |Digital Output|`robot.OUTPUT`|Allows you to write a high or low signal|
 |Digital Input|`robot.INPUT`|Allows you to read a high or low signal|
 |Analog Input|`robot.INPUT_ANALOG`|Allows you to read a voltage, like a voltmeter|
-|Pullup Input|`robot.INPUT_PULLUP`|Like analog, but uses a [weak pullup resistor](/gpio.html#pull-ups)|
+|Pullup Input|`robot.INPUT_PULLUP`|Like input, but uses a [weak pullup resistor](/gpio.html#pull-ups)|
```

Comment:
Might be nice to give an example of what uses a pull up resistor, as appears a little confusing to those who have never done any electronics before!

Maybe add an examples column? For a pull up I think this is used for buttons but can't be sure!